### PR TITLE
added IBM vendor and corresp. manager logos

### PR DESCRIPTION
**Pull request by iv10**
- IBM vendor logo
- IBM Cloud Manager logo (symlink to vendor logo)
- IBM Network Manager logo (symlink to vendor logo)
- IBM Storage Manager logo (symlink to vendor logo)

### CODE REVIEW

**Comment by iv10** [comment]
Hi, everyone. Please merge this as we pushed our IBM Cloud plugin to the public Github yesterday.
See here https://github.com/ManageIQ/manageiq-providers-ibm_cloud.

PS. In the future I'll be working from my new IBM public Github account iv1111.

File: app/assets/images/svg/vendor-ibm.svg [hunk 1]
```diff
@@ -0,0 +1,371 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="256px" height="256px" viewBox="0 0 256 256" enable-background="new 0 0 256 256" xml:space="preserve">  <image id="image0" width="256" height="256" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQAAAAEACAIAAADTED8xAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```
Comment (skateman) [inline app/assets/images/svg/vendor-ibm.svg:4]: As I see this is an inlined png image into a svg file. I would prefer it a vector version if you have it. Of course if it's not available, then we can go with this one.

**Comment by iv10** [comment]
the cloud manager logo should stay plural (has to do with our branding), rest singular, plus changed logo as suggested by @skateman

**Comment by iv10** [comment]
Hi, i removed the WIP in title, so we're good to go. If everything is ok, then let's merge this. Thanks.

File: app/assets/images/svg/vendor-ibm.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...-5.19L69.82,22a5.19,5.19,0,0,0-9,5.19Z"/><path class="cls-2" d="M137.89,39.15A5.19,5.19,0,0,0,143.08,34V5.19a5.2,5.2,0,0,0-10.39,0V34A5.19,5.19,0,0,0,137.89,39.15Z"/><path class="cls-2" d="M193.48,54.05a5.2,5.2,0,0,0,7.09-1.91L215,27.23A5.19,5.19,0,0,0,206,22L191.58,47A5.19,5.19,0,0,0,193.48,54.05Z"/><path class="cls-2" d="M238.67,97.34a5.23,5.23,0,0,0,2.6-.7l24.91-14.38a5.2,5.2,0,1,0-5.2-9L236.07,87.65a5.19,5.19,0,0,0,2.6,9.69Z"/><path class="cls-2" d="M218.34,102.31A94.52,94.52,0,0,0,204.11,84.1,93.71,93.71,0,0,0,47.87,124.35,60.48,60.48,0,0,0,60,244H204.24a71.54,71.54,0,0,0,14.1-141.68ZM79,91.45a83.27,83.27,0,0,1,117.76,0,84.59,84.59,0,0,1,8.16,9.46h-.16a71.55,71.55,0,0,0-56.22,116.46l8.08-6.53a61.16,61.16,0,0,1,48.06-99.55,62.56,62.56,0,0,1,7,.46,83.3,83.3,0,0,1-73.8,121.85h-.06a83.27,83.27,0,0,1-83.24-83.27,84,84,0,0,1,1.68-16.75c1.38-.11,2.77-.17,4.18-.17h.38a50.16,50.16,0,0,1,49.72,50.47L121,184a60.55,60.55,0,0,0-60-60.93h-.45l-1.3,0A82.76,82.76,0,0,1,79,91.45ZM60.09,233.6a50.1,50.1,0,0,1-14.73-97.86,95.49,95.49,0,0,0-1.13,14.59,93,93,0,0,0,27.43,66.22,94,94,0,0,0,23.27,17Zm144.15,0h-23.4a93.86,93.86,0,0,0,43.74-118.82A61.15,61.15,0,0,1,204.24,233.6Z"/></g></g></g></g></svg>
```
Comment (skateman) [inline app/assets/images/svg/vendor-ibm.svg:1]: There's still a png encoded in the svg file :confused: 
```
xlink:href="data:image/png;base64
```

Comment (iv1111) [reply app/assets/images/svg/vendor-ibm.svg:1]: ok, let me check

Comment (iv1111) [reply app/assets/images/svg/vendor-ibm.svg:1]: I've totally forgotten. I've spoken to Jay and as it seems, we're currently not able to get an officially sanctioned **vector** image from our branding team. The idea is to submit this one until we get a vector image in the future.

Comment (skateman) [reply app/assets/images/svg/vendor-ibm.svg:1]: Okay, just create a new PR replacing this file if you get your hands on a proper vector one. I'll merge this after it's green...

**Review by skateman — APPROVED** [review]
The Seal of Approval

**Comment by skateman** [comment]
Ugh, maybe one thing. Can you squash your commits to a single one to make it look nicer?

**Comment by iv10** [comment]
I squashed my commits.

**Pull request merged**